src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/curatedcontent/CuratedEditorFormPageObject.java

```diff
@@ -0,0 +1,48 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent;
+
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.BasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.curatedcontent.imageupload.UploadImageModalComponentObject;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * @ownership: Content X-Wing
+ */
+public class CuratedEditorFormPageObject extends BasePageObject {
```

Comment:
This class represents the Editor Form, and serves as parent class for Item, Section, and Category form

EditorHomePageObject is responsible for the main editor view